### DISCUSSION TITLE
Best way to cache?

### DISCUSSION THREAD

**Discussion by studiopatagon** [jeremyevans/roda#311] [General]
Is there a way to cache a specific route/ view's database query ? Example, I have a blog index and I don't need it to query the DB on every page load . . . I'm using Roda + Sequel.

**Comment by opya** [comment]
@willowlight , did you try [Roda::RodaPlugins::Caching](https://roda.jeremyevans.net/rdoc/classes/Roda/RodaPlugins/Caching.html)

**Reply by studiopatagon** [reply]
My bad, I searched for "cache" instead of caching on the Roda documentation page 🤣 Thanks so much!
